PlotMarkersHandler.cls

```diff
@@ -0,0 +1,88 @@
+public with sharing class PlotMarkersHandler {
+    public static void plotMarkersMethodForAccountAndContact(Map<Id,sObject> mapForFetchingIds)
+    {
+           googleCallout(mapForFetchingIds.keyset());
+    }
+    @future(callout= True)
+    public static void googleCallout(Set<id> newRecordId1)
+    {
+        List<sObject> parametersList = new  List<sObject>();
+        List<Id> newRecordId = new list<Id>();
+        newRecordId.addAll(newRecordId1);
+        List<double> locationList = new List<double>();
+        String searchQuery;
+        List<String> listofLinks;
+        String googleMapEndPointUrl ='https://maps.googleapis.com/maps/api/geocode/json?sensor=false&address=';
+        if(String.valueOf(newRecordId[0]).substring(0,3) == '001')
+        {
+             parametersList = [Select id, Name ,BillingAddress ,BillingLatitude,BillingLongitude from Account where id in : newRecordId];
+        }
+        else
+        {
+             parametersList = [Select id, Name ,MailingAddress ,MailingLatitude,MailingLongitude from Contact where id in : newRecordId];
+        }
+                                           
+        for(sObject sObjectObj : parametersList ){   
+             Address addr;
+           
+            if(String.valueOf(sObjectObj.id).substring(0,3) == '001')
+            {     
+                Account convertedAccount = (Account)sObjectObj;
+                addr = convertedAccount.BillingAddress;
```

Comment:
You could have do like `sObjectObj.get('BillingAddress')` . So that you don't need if condition to fetch address. This can be combine with ternary operator like below
```
if(String.valueOf(newRecordId[0].getSObjectType()).equalsIgnoreCase(String.valueOf(Account.sObjectType)))
? sObjectObj.get('BillingAddress')
: sObjectObj.get('MailingAddress')
```